### ISSUE TITLE
Lower level brand logo and theme access

### ISSUE THREAD

**Issue by powerpbx** [irontec/ivozprovider#889]
We envision a scenario where it would be nice to have brand logo and theme changes accessible by the brand operator.  Have you considered that possibility?

**Comment by Kaian** [comment]
Yes, we considered some basic information of Brand and Company tables should be accessible by their owners.
Things like invoicing data, language or default DDI should be available to Company admins, so they actually know what _use Company's default_ means in multiple screens.

Sadly this is not easy to do with current interface framework. Klear excels at drawings tables and rows, but it does not so well on drawing a single edit screen.

We can create new tables (BrandSettings and CompanySettings) and make its data accessible to brand and company Administrators.

We are also currently working on new administration portals,  so a new approach may arise.

So, yes. We considered it and we agree, but I don't know when this will be available. Moving to 2.x.
